catalog/task/collect-data/0.1/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Install the CRDs so we can create/get them
+.github/scripts/install_crds.sh
```

Comment:
I'm fine with this. But I wonder if it would make sense to move these resources to the same directory as the test that needs it. Maybe let's keep it this way for now and if it grows in the future and there's too much stuff, we can reconsider? WDYT?

Reply:
Yeah, I figured that installing our CRDs was generic enough to not have it in the task's tests directory. Any task that interacts with our CRDs will need it, so I can definitely see another one needing it in the future (although no others do now afaik)

Reply:
OK, sounds good